build.sbt

```diff
@@ -25,7 +25,6 @@ scalacOptions ++= Seq(
   "-encoding", "UTF-8",       // yes, this is 2 args
   "-feature",
   "-unchecked",
-  "-Xfatal-warnings",
```

Comment:
Original intent is to guard certain level of quality. This comes from a commercial project I've been working on where such set of flags does a good job. Do you see any concerns regarding this flag?